src/osdc/Objecter.cc

```diff
@@ -2460,6 +2460,10 @@ void Objecter::_op_submit(Op *op, shunique_lock& sul, ceph_tid_t *ptid)
   sl.unlock();
   put_session(s);
 
+ if ((cct->_conf->rgw_fastfail_homeless_timeout > 0) && s->is_homeless()) { 
+     uint64_t t = timer.add_event(ceph::make_timespan(cct->_conf->rgw_fastfail_homeless_timeout), [this, tid]() {_op_cancel(tid, -ETIMEDOUT);});
+     ldout(cct, 0) << __func__ << " Op cancellation due to homeless session. t =" << t << dendl;
```

Comment:
this message should only be printed when the timer event fires (so inside the lambda)